### ISSUE TITLE
Question of saving and evaluating models

### ISSUE THREAD

**Issue by SZUHvern** [funnyzhou/REFERS#6]
Dear Authors,

I can not find the code of saving models in Pre-train task, also, there are no metrics to evaluate the model except for loss. 
Could you mind add the code to save the model and provide some suggestion to help me in evaluating the model?

**Comment by funnyzhou** [comment]
This is a good question.

As far as I'm concerned, there is no metric to evaluate the model performance during the pre-training stage. Actually, the common practice is to adjust the number of pre-training epochs and evaluate associated checkpoints on the downstream tasks.

**Comment by SZUHvern** [comment]
Thanks!
